docs/tutorials/tphi_characterization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
I can't review the tutorial. Something's wrong with indent font. I'll review after it's fixed. Note that in Latex we're talking about `\varphi` and not `\phi`.